srcpkgs/grandorgue/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'grandorgue'
+pkgname=grandorgue
+version=3.13.0
+revision=1
+archs="x86_64"
+build_style="cmake"
+configure_args="-DCMAKE_BUILD_TYPE=Release -G Ninja"
+hostmakedepends="pkg-config git ImageMagick"
+makedepends="jack-devel eudev-libudev-devel zlib-devel yaml-cpp-devel wxWidgets-devel wxWidgets-common-devel wxWidgets-gtk3-devel fftw-devel wavpack-devel alsa-lib-devel"
+depends="gettext po4a zip"
+short_desc="Pipe Organ Simulator"
+maintainer="Nicolo Davis <nicolodavis@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/GrandOrgue/grandorgue"
+
+export WX_CONFIG=wx-config-gtk3
+
+do_fetch() {
+	git clone --recurse-submodules --depth 1 --branch ${version}-${revision} https://github.com/GrandOrgue/grandorgue.git ${wrksrc}
```

Comment:
do not use `git clone` to fetch. if a packages has submodules, they should be added as distfiles. see the `edk2-ovmf` template for an example.